delete_duplicates.py

```diff
@@ -0,0 +1,13 @@
+"""
+With a given list [12,24,35,24,88,120,155,88,120,155], write a program to
+print this list after removing all duplicate 
+values with original order reserved.
+"""
+
+
+def original_numbers(numbers):
+    return list(set(numbers))[::-1]
+
```

Comment:
Too many blank lines.

Please try to follow PEP-8 recommendations.

Reply:
done